src/components/grant-details-page/grant-details-tabs/GrantDetailsTabs.js

```diff
@@ -51,7 +51,7 @@ export function GrantDetailsTabs({ grant, filteredOutTabs }) {
                   ? grant.grantSupportingDocuments
                   : [];
                 return (
-                  <GrantDetailsTabContent tab={tab} index={index}>
+                  <GrantDetailsTabContent tab={tab} index={index} key={index}>
```

Comment:
probably not a massive issue because its server-side rendered but indices shouldn't be used as keys